+nosnoc/+time_freezing/cls_elastic.m

```diff
@@ -11,8 +11,8 @@
     end
     
     dims.n_contacts = length(cls_model.f_c);
-    if dims.n_contacts > 1
-        error("nosnoc: time freezing for elastic contacts currently only supports a single contact.")
+    if dims.n_c > 1
+        error("nosnoc:time_freezing:cls_elastic:too_many_contacts", "nosnoc: time freezing for elastic contacts currently only supports a single contact.")
```

Comment:
i am curious can this "path" from which class-function the error'warrning comes be automated instead of the manual typing

Reply:
:disappointed: I was hoping this could be done too. By default the answer is no but there _are_ ways to get the stack at runtime so I can write a little utility, say `nosnoc_error` which could wrap `error` and probably get what we want.  I did some hacking around with this in `vdx` previously so let me see what I can do.